src/AppBundle/Controller/AutomatedEditsController.php

```diff
@@ -67,21 +68,30 @@ public function indexAction(Request $request)
             );
         }
 
-        // set default wiki so we can populate the namespace selector
+        /** @var ApiHelper */
+        $api = $this->get("app.api_helper");
+
+        // Set default project so we can populate the namespace selector.
         if (!$project) {
             $project = $this->container->getParameter('default_project');
         }
 
-        /** @var ApiHelper */
-        $api = $this->get("app.api_helper");
+        $projectData = ProjectRepository::getProject($project, $this->container);
+
+        // Default values for the variables to keep the template happy
+        $namespaces = null;
+
+        // If the project exists, actually populate the values
+        if ($projectData->exists()) {
+            $namespaces = $projectData->getNamespaces();
+        }
 
-        // TODO: add namespace support
         return $this->render('autoEdits/index.html.twig', [
             'xtPageTitle' => 'tool-autoedits',
             'xtSubtitle' => 'tool-autoedits-desc',
             'xtPage' => 'autoedits',
             'project' => $project,
-            // 'namespaces' => $api->namespaces($project),
+            'namespaces' => $namespaces,
```

Comment:
I don't believe per-namespace support is there yet but I'm going to add it soon, so no harm in uncommenting